gmso/utils/ff_utils.py

```diff
@@ -312,8 +312,12 @@ def _validate_schema(xml_path_or_etree, schema=None):
     ff_xml = xml_path_or_etree
     if not isinstance(xml_path_or_etree, etree._ElementTree):
         ff_xml = etree.parse(xml_path_or_etree)
-
-    xml_schema.assertValid(ff_xml)
+    try:
+        xml_schema.assertValid(ff_xml)
+    except:
+        raise ForceFieldParseError(
+            f"xml scheme {xml_schema} has invalid elements. Check the sections of the file."
+        )
```

Comment:
What error message would the `xml_schema.assertValid` raised? Because the alternative message on the `except` doesn't provide that much info either.

Reply:
Added a more descriptive error, sourced from the error handling Foyer uses when the xml is not generated properly. Although having looked at it, ff_utils module is going to be totally out of use as well once `convert_from_foyer_xml` is deprecated soon. 

So some of these errors are going to have to be tested for on that end, not in GMSO. For now though it's here once this PR is merged.

Reply:
Should raise the error handling that is raised by lxml, which will give the section in question that is not matching the force field schema.
```raise ForceFieldParseError((error_text, ex, line))```